read-fonts/src/tables/glyf.rs

```diff
@@ -610,7 +610,9 @@ pub fn to_path(
     fn to_f32(x: F26Dot6) -> f32 {
         x.to_f64() as f32
     }
-    const TWO: F26Dot6 = F26Dot6::from_i32(2);
+    fn midpoint(a: Point<F26Dot6>, b: Point<F26Dot6>) -> Point<F26Dot6> {
```

Comment:
this is obtuse enough I suggest a comment. bonus points for a citation.

Reply:
The relevant code is here: https://github.com/freetype/freetype/blob/de8b92dd7ec634e9e2b25ef534c54a3537555c11/src/base/ftoutln.c#L123

I’ll add a note and include the link.